webapp/services/get_user.py

```diff
@@ -0,0 +1,36 @@
+import webapp2
+import json
+import logging
+from google.appengine.ext import ndb
+from google.appengine.api import search
+from models.user import User
+
+class GetUser(webapp2.RequestHandler):
+
+    def get(self):
+
+		user_id = int( self.request.get('user_id', 0) )
+
+		user = User.get_by_id(user_id)
+
+		if user != None:
+			user_creation_date = user.creation_date
```

Comment:
You can do it in one line, no need for extra variables. :)
`user_creation_date = user.creation_date.strftime('%m/%d/%Y')`